lib/Myriad/Config.pm

```diff
@@ -45,9 +45,10 @@ our %DEFAULTS = (
     library_path           => '',
     opentracing_host       => 'localhost',
     opentracing_port       => 6832,
-    subscription_transport => 'perl',
-    rpc_transport          => 'perl',
-    storage_transport      => 'perl',
+    subscription_transport => undef,
+    rpc_transport          => undef,
+    storage_transport      => undef,
+    myriad_transport       => 'perl',
```

Comment:
How about calling this `default_transport` so that when generating env variables to check down below it gets seen as `MYRIAD_DEFAULT_TRANSPORT`?